backend/controllers/receptionMove/confirmReceptionMove.ts

```diff
@@ -98,6 +98,9 @@ export default ({ prisonApi, whereaboutsApi }) => {
         commentText: comment,
       })
 
```

Comment:
This will flush Flash of any form values (radios selected, text boxes completed) so they can't be accessed later